### ISSUE TITLE
Crash with Error - WaterMoveUnit is not defined.

### ISSUE THREAD

**Issue by XenoForce** [z0mbiesrock/Diamond-Ore#51]
Date: 09 July 2023

Mindustry crashes when I install the Diamond Ore mod.

It crashes with the following error message:

"WaterMoveUnit" is not defined.

Request: Please could you fix this error for us.

This is such a cool mod. It deserves to live on in Mindustry v7.

The units are very original.

I played Diamond Ore mod in the past, and I really liked it. So, please, fix it for us. Thank you.

**Comment by OmniNova** [comment]
Like the mod, but same problem, plz fix.

**Comment by Vistup** [comment]
Yeah, I love this mod too, but doesn't work now because of this...Please Fix ASAP

**Comment by traincrisis** [comment]
I have the same problem

**Comment by chinhonnang0000** [comment]
same problem here with v146.1
